ipaserver/plugins/hbacrule.py

```diff
@@ -130,14 +115,15 @@ class hbacrule(LDAPObject):
     container_dn = api.env.container_hbac
     object_name = _('HBAC rule')
     object_name_plural = _('HBAC rules')
-    object_class = ['ipaassociation', 'ipahbacrule']
-    permission_filter_objectclasses = ['ipahbacrule']
+    object_class = ['ipaassociation']
```

Comment:
Maybe we should explicitly say in design, that new HBACrulesV2 will not be (or should not be) accessible from old servers, modifications are able only on new servers